tools/prow-auto-bumper/pullrequest.go

```diff
@@ -0,0 +1,161 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"strings"
+
+	"github.com/google/go-github/github"
+	"github.com/knative/test-infra/shared/ghutil"
+)
+
+func makeCommitSummary(vs versions) string {
+	return fmt.Sprintf("Update prow from %s to %s, and other images as necessary.", vs.oldVersion, vs.newVersion)
+}
+
+func getMatchTitle() string {
+	return "Update prow from"
+}
+
+func generatePRBody(extraMsgs []string) string {
+	var body string
+	if len(extraMsgs) > 0 {
+		body += "Warnings:\n"
+		for _, msg := range extraMsgs {
```

Comment:
Consider using [stringbuilder](https://www.calhoun.io/concatenating-and-building-strings-in-go/)?

Reply:
This loop would rarely be executed, so this should be fine